### PR TITLE
Move Schools from markdown to yml file

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**
https://github.com/HSF/hsf.github.io/issues/651

### CODE REVIEW

**Comment by klieret** [comment]
Hi, but this still needs a change in ``list_of_schools.md``, right?

File: Schools/_posts/2020-06-07-tCSC.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Thematic CERN School of Computing 2020
```
Comment (klieret) [inline Schools/_posts/2020-06-07-tCSC.md]: shouldn't this also go to the yml?

Comment (GuillermoFidalgo) [reply Schools/_posts/2020-06-07-tCSC.md]: I believe it already is in the yml

**Comment by GuillermoFidalgo** [comment]
> Thanks a lot! This still needs a change in `list_of_schools.md` though, right?

Working on this now. Thanks for reminding me

**Comment by GuillermoFidalgo** [comment]
Thanks for the reminder. I will get to it today.

**Comment by klieret** [comment]
Thanks a lot for your work! :)

**Pull request merged**